unit/util/ieee_float.test.cpp

```diff
@@ -0,0 +1,33 @@
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <util/ieee_float.h>
+
+TEST_CASE("ieee float can handle 1", "[core][util][ieee_floatt]")
+{
+  int one = 1;
+  double d_one = 1;
+  ieee_floatt ieee_one(ieee_float_spect(52, 11));
+
+  SECTION("Basic context")
+  {
+    REQUIRE(std::isnormal(one));   // Holds
```

Comment:
Appearently Windows didn't implement the overloads (A) in <https://en.cppreference.com/w/cpp/numeric/math/isnormal>. Maybe just drop this requirement, it's implicitly checked by the version with `d_one` anyway.